networking_generic_switch/devices/corsa_devices/__init__.py

```diff
@@ -207,34 +225,142 @@ def plug_port_to_network(self, port, segmentation_id):
+
         try:
-          with ngs_lock.PoolLock(self.locker, **self.lock_kwargs):
-            # Make sure the tunnel_mode is 'passthrough'    
-            corsavfc.port_modify_tunnel_mode(headers, url_switch, port_num, 'passthrough') 
+            with ngs_lock.PoolLock(self.locker, **self.lock_kwargs):
+                # Make sure the tunnel_mode is 'passthrough'                                                                                                                                                                                                       
+                corsavfc.port_modify_tunnel_mode(headers, url_switch, port_num, 'passthrough')
 
-            # get the bridge from segmentation_id
-            br_id = corsavfc.get_bridge_by_segmentation_id(headers, url_switch, segmentation_id)
+                # get the bridge from segmentation_id                                                                                                                                                                                                              
+                br_id = corsavfc.get_bridge_by_segmentation_id(headers, url_switch, segmentation_id)
 
-            try:
-                # unbind the port (probably not necessary)
-                # openflow port was mapped to the physical port with the same port number in plug_port_to_network
-                ofport = self.get_ofport(br_id,port)
-                corsavfc. bridge_detach_tunnel(headers, url_switch, br_id, ofport)
-                LOG.info("needed to delete_port: probably a leaked port from a node that did not completely boot the previous time. ")
-            except Exception as e:
-                LOG.info("Tried to delete_port but it was not there: this is expected." + traceback.format_exc())
-                pass
-
-            # bind the port
-            # physical port is mapped to the openflow port with the same port number
-            ofport = self.get_ofport(br_id,port)
-            corsavfc.bridge_attach_tunnel_passthrough(headers, url_switch, br_id, port_num, ofport, tc = None, descr = None, shaped_rate = None)
+                corsavfc.bridge_attach_tunnel_passthrough(headers, url_switch, br_id, port_num, ofport, tc = None, descr = None, shaped_rate = None)
 
         except Exception as e:
             LOG.error("Failed to plug to network: " + str(traceback.format_exc()))
             raise e
- 
-    def delete_port(self, port, segmentation_id):
+
+
+
+    def __bind_ctag_tunnel(self, segmentation_id, node_vlan, dst_switch_name):
+        LOG.info("adding to vfc_host: __bind_ctag_tunnel")      
+        LOG.info("PRUTH: switch: " + self.config['name'] + ", VFCHost: " + str(segmentation_id) + ", " + str(node_vlan) + ", " + str(dst_switch_name))
+
+        #port_num=port[2:]
+        token = self.config['token']
+        headers = {'Authorization': token}
+
+        logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
```

Comment:
This overrides the base logging configuration, which is global - if it is going to be used, it should be placed prominently at the top of the file.

Reply:
I don't think this is necessary.  Mert, do you know why this is here?

Reply:
No. I did not add that line.

Reply:
I suspect it can be removed then.